index.js

```diff
@@ -0,0 +1,26 @@
+import { Books } from './modules/books.js';
+import { setupNavigation } from './modules/navigation.js';
+import dt from './modules/datetime.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  setupNavigation();
+  const a = new Books();
+  a.createObject();
+  a.displayBooks();
+  a.saveToLocalStorage();
+
+  document.getElementById('date').innerHTML = dt;
+
+  document.getElementById('addbtn').addEventListener('click', () => {
+    a.addBooks();
+  });
+
+  // Remove button event listener
+  document.addEventListener('click', (event) => {
+    if (event.target.classList.contains('remove-btn')) {
+      // eslint-disable-next-line radix
```

Comment:
- [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

Reply:
Thank you Abayomi, I gave attention to these problems!